agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4306,8 +4307,9 @@ bool slave_thread::handle_client_association_request(Socket *sd, ieee1905_1::Cmd
             return false;
         }
 
-        request_out->mac()   = sta_mac;
-        request_out->bssid() = bssid;
+        request_out->mac()             = sta_mac;
+        request_out->bssid()           = bssid;
+        request_out->validity_period() = validity_period;
```

Comment:
Why do we need an additional variable here?
```suggestion
        request_out->validity_period() = association_control_request_tlv->validity_period_sec();
```